### PR TITLE
Changes to make service connection optional for CLT Task

### PR DESCRIPTION
**Pull request by vinitsuri**
- Changes to make the service connection optional for the CLT Build task

### CODE REVIEW

File: Tasks/RunLoadTest/Start-CloudLoadTest.ps1 [hunk 1]
```diff
@@ -1,480 +1,196 @@
...
+		$thresholdImage="bowtie-status-success"
 	}
 	
 	
-    $summary = ('<span class="bowtie-icon {3}" />   {4}<br/>[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUri , $run.name, $thresholdImage, $thresholdMessage)
-    
-	('<p>{0}</p>' -f $summary) | Out-File  $summaryFile -Encoding ascii -Append
-    UploadSummaryMdReport $summaryFile
+	$summary = ('<span class="bowtie-icon {3}" />   {4}<br/>[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUri , $run.name, $thresholdImage, $thresholdMessage)
+	('<p>{0}</p>' -f $summary) >>  $summaryFile
```
Comment (dpksinghal) [inline Tasks/RunLoadTest/Start-CloudLoadTest.ps1]: > $summ [](start = 31, length = 6)

same here.

File: Tasks/RunLoadTest/task.json [hunk 2]
```diff
@@ -1,83 +1,82 @@
...
+    },
+    {
+      "name": "MachineType",
+      "type": "radio",
+      "label": "Run load test using",
+      "required": true,
+      "defaultValue": "0",
+      "options": {
+        "2": "Self-provisioned agents",
+        "0": "Automatically provisioned agents"
```
Comment (dpksinghal) [inline Tasks/RunLoadTest/task.json]: >0 [](start = 9, length = 1)

Change it to 1

**Comment by dpksinghal** [comment]
:clock1:

File: Tasks/RunJMeterLoadTest/Start-ApacheJMeterTest.ps1 [hunk 3]
```diff
@@ -1,371 +1,164 @@
...
+	Remove-Item $resultsMDFolder\$resultFilePattern -Exclude $excludeFilePattern -Force
+	$summaryFile =  ("{0}\ApacheJMeterTestResults_{1}_{2}_{3}_{4}.md" -f $resultsMDFolder, $env:AGENT_ID, $env:SYSTEM_DEFINITIONID, $env:BUILD_BUILDID, $run.id)
 
-    $summary = ('[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUrl , $run.name)
-     	
-	('<p>{0}</p>' -f $summary) | Out-File  $summaryFile -Encoding ascii -Append
-    UploadSummaryMdReport $summaryFile
+	$summary = ('[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUrl , $run.name)
+	
+	('<p>{0}</p>' -f $summary) >>  $summaryFile
```
Comment (vinitsuri) [inline Tasks/RunJMeterLoadTest/Start-ApacheJMeterTest.ps1]: resolved

Comment (dpksinghal) [inline Tasks/RunJMeterLoadTest/Start-ApacheJMeterTest.ps1]: it is not yet resolved.

---
In reply to: [88878625](https://github.com/vinitsuri/vsts-tasks/pull/1#discussion_r88878625) [](ancestors = 88878625)

**Pull request merged**